src/tools/cephfs/top/cephfs-top

```diff
@@ -209,6 +209,95 @@ class FSTop(object):
             raise FSTopException('Cannot handle unknown metrics from'
                                  f'\'ceph fs perf stats\': {missing}')
 
+    def dump_values_to_stdout(self):
```

Comment:
Hmmm.. this is a _bit_ off-putting. Could we move the relevant bits into a separate source?

Reply:
@vshankar 
We could do that. But then we should move this part also to the new source: https://github.com/ceph/ceph/blob/main/src/tools/cephfs/top/cephfs-top#L29-L107. Because both `dump` and `ncurses` code needs those definitions. Please confirm.

Reply:
https://github.com/ceph/ceph/blob/main/src/tools/cephfs/top/cephfs-top#L29-L157
it is.

Reply:
Yeh, I think, it would be better if anything related to display (UI or terminal) can be maintained in a separate source.

Reply:
You want to maintain these definitions [1] common or maintain them separately (separate definitions) for UI and terminal? 

https://github.com/ceph/ceph/blob/main/src/tools/cephfs/top/cephfs-top#L29-L157

Reply:
Ideally, definitions that are used in both display formats should be kept common.

Reply:
Alright, I have a new code based on that and I'll push soon after the cleanup.